llvm/test/CodeGen/AArch64/and-sink.ll

```diff
@@ -1,3 +1,4 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
```

Comment:
Please submit this change separately.
I'm not sure if this test case is broken.